backend/app/routes/admin/getAdmins.js

```diff
@@ -0,0 +1,52 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+const getAdminsAggregate = (adminType, page) => {
+  const pipeline = [
+    {
+      $project: {
+        _id: 0,
+        firstName: 1,
+        lastName: 1,
+        username: 1,
+        email: 1,
+        contact: 1,
+        isSuperAdmin: 1,
+      },
+    },
+  ];
+  if (adminType === 'superAdmin' || adminType === 'admin') {
+    pipeline.unshift({
+      $match: {
+        isSuperAdmin: adminType === 'superAdmin',
+      },
+    });
+  }
+  if (page === undefined) page = 1;

```

Comment:
Please run linter before committing, this line is giving an error

Reply:
> Please run linter before committing, this line is giving an error.
> Please fix it

So sorry :( I forgot to run it
Can we have an issue to run linting as pre commit hook please ?